### PR TITLE
[release-1.3]cherry-pick into 1.3: Add/Remove service from mesh

### PR DESCRIPTION
**Pull request by irisdingbj**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[x] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by irisdingbj** [comment]
@sdake @linsun Is it possible to let this PR part of release 1.3?

File: istioctl/cmd/remove-from-mesh.go [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+	"istio.io/istio/istioctl/pkg/util/handlers"
+	"istio.io/pkg/log"
+
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func removeFromMeshCmd() *cobra.Command {
+	addToMeshCmd := &cobra.Command{
```
Comment (rlenglet) [inline istioctl/cmd/remove-from-mesh.go]:
```suggestion
	removeFromMeshCmd := &cobra.Command{
```

File: istioctl/cmd/remove-from-mesh.go [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+		Use:     "remove-from-mesh",
+		Aliases: []string{"remove"},
```
Comment (rlenglet) [inline istioctl/cmd/remove-from-mesh.go]: This alias seems to too general to me to claim for this command.

File: istioctl/cmd/add-to-mesh.go [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+	IP             []string
+	PortList       model.PortList
+	Labels         map[string]string
+	Annotations    map[string]string
+}
+
+func addToMeshCmd() *cobra.Command {
+	addToMeshCmd := &cobra.Command{
+		Use:     "add-to-mesh",
+		Aliases: []string{"add"},
```
Comment (rlenglet) [inline istioctl/cmd/add-to-mesh.go]: This alias seems to too general to me to claim for this command.

**Comment by rlenglet** [comment]
And @irisdingbj please add an entry for those commands in the [1.3 Release Note](https://github.com/istio/istio/wiki/Release-Note-1.3-Draft).

**Comment by rlenglet** [comment]
Test failures don't seem related to this PR.

**Comment by irisdingbj** [comment]
> And @irisdingbj please add an entry for those commands in the [1.3 Release Note](https://github.com/istio/istio/wiki/Release-Note-1.3-Draft).

Thanks @rlenglet I have added these two commands into our release note

**Comment by irisdingbj** [comment]
> We shouldn't backport new features at this stage, but this one is OK since it's self-contained and experimental.
> I reviewed in more details and have a few nitpicks, but they're not blocking this PR. You may want to address those in another PR later.

sure.

**Pull request merged**